src/Components/edit/Edit.tsx

```diff
@@ -26,7 +29,9 @@ const Edit = ({ toast, setEditVisibility, employee }: EditProps) => {
       if (employee !== editFormik.initialValues) {
         updateMutation.mutate(employee);
       } else {
-        showInfoMessage(toast);
+        const messageSummary = "No changes were made.";
+        const message = "In order to update information you should change field values.";
+        showMessage(toast, "info", messageSummary, message);
```

Comment:
do not use showMessage, consider it as private method. create method like showInfo(messageSummary, message), showWarning .. Inside showInfo call showMessage with corresponding params

Reply:
Fixed.